test/integration/ads_integration_test.cc

```diff
@@ -1359,8 +1442,8 @@ TEST_P(AdsClusterV2Test, CdsPausedDuringWarming) {
 
   // Send the second warming cluster.
   sendDiscoveryResponse<envoy::config::cluster::v3::Cluster>(
-      cds_type_url, {buildCluster("warming_cluster_2")}, {buildCluster("warming_cluster_2")}, {},
-      "3", true);
+      cds_type_url, {buildCluster("warming_cluster_1"), buildCluster("warming_cluster_2")},
+      {buildCluster("warming_cluster_1"), buildCluster("warming_cluster_2")}, {}, "3", true);
   test_server_->waitForGaugeEq("cluster_manager.warming_clusters", 2);
   // We would've got a Cluster discovery request with version 2 here, had the CDS not been paused.
 
```

Comment:
This is the correct CDS response when we want to send (additional) second warming cluster.